### PR TITLE
Upgrade gradle to 6.7.1, plus latest available dependencies updated

### PR DESCRIPTION
**Pull request by chris-ryan-square**

### CODE REVIEW

File: dependencies.gradle [hunk 1]
```diff
@@ -21,13 +20,13 @@ ext.dep = [
   "grpcNetty": "io.grpc:grpc-netty:1.21.0",
   "grpcProtobuf": "io.grpc:grpc-protobuf:1.21.0",
   "grpcStub": "io.grpc:grpc-stub:1.21.0",
-  "guava": "com.google.guava:guava:30.0-jre",
-  "guavaTestLib": "com.google.guava:guava-testlib:30.0-jre",
+  "guava": "com.google.guava:guava:29.0-jre",
```
Comment (swankjesse) [inline dependencies.gradle:23]: Downgrade?

File: dependencies.gradle [hunk 2]
```diff
@@ -89,19 +88,19 @@ ext.dep = [
...
+  "retrofitMoshi": "com.squareup.retrofit2:converter-moshi:2.9.0",
+  "retrofitProtobuf": "com.squareup.retrofit2:converter-protobuf:2.9.0",
+  "retrofitWire": "com.squareup.retrofit2:converter-wire:2.9.0",
   "servletApi": "javax.servlet:javax.servlet-api:3.1.0",
-  "shadowJarPlugin": "com.github.jengelman.gradle.plugins:shadow:5.1.0",
+  "shadowJarPlugin": "com.github.jengelman.gradle.plugins:shadow:6.1.0",
   "slf4jApi": "org.slf4j:slf4j-api:1.7.28",
   "tink": "com.google.crypto.tink:tink:1.2.0",
-  "tracingDatadog": "com.datadoghq:dd-trace-api:0.67.0",
+  "tracingDatadog": "com.datadoghq:dd-trace-api:0.65.0",
```
Comment (swankjesse) [inline dependencies.gradle:99]: Downgrade?

**Review by swankjesse — APPROVED** [review]

File: dependencies.gradle (hunk 2 above)
Comment (adrw) [reply dependencies.gradle:99]: Seems like these need to be updated in `master-dependencies.json` for the dep-upgrade to not downgrade.

File: dependencies.gradle (hunk 1 above)
Comment (chris-ryan-square) [reply dependencies.gradle:23]: Good catch - I was just using the dep-upgrade and reverting deps that caused build problems.  I'll revert these and retry the build.

File: dependencies.gradle (hunk 2 above)
Comment (chris-ryan-square) [reply dependencies.gradle:99]: See above

**Comment by chris-ryan-square** [comment]
Not going to do these changes for now.  Will revisit later.

**Pull request closed without merging**